app/featureset/status_string.go

```diff
@@ -10,19 +10,30 @@ func _() {
 	// An "invalid array index" compiler error signifies that the constant values have changed.
 	// Re-run the stringer command to generate them again.
 	var x [1]struct{}
+	_ = x[enable-9223372036854775807]
+	_ = x[disable-0]
```

Comment:
What is the purpose of those two? Disable I can understand - if we want to disable the feature later on, but enable?

Reply:
That file is auto generated by stringer